cli/assets/README.md

```diff
@@ -0,0 +1,59 @@
+# Foundry
+
+Foundry is a smart contract development toolchain.
```

Comment:
I'd expand this section to include what the repo readme says about the parts of foundry (forge, cast, anvil)

Reply:
```
Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.

Foundry consists of:

-   Forge: Ethereum testing framework (like Truffle, Hardhat and DappTools).
-   Cast: Swiss army knife for interacting with EVM smart contracts, sending transactions and getting chain data.
-   Anvil: Local Ethereum node, akin to Ganache, Hardhat Network.
-   Chisel: Fast, utilitarian, and verbose solidity REPL.
```

This is what I'm planning to add, let me know your thoughts.

Reply:
Looks good!